frontends/election-manager/src/components/election_manager_writein_tally_report.tsx

```diff
@@ -0,0 +1,202 @@
+import {
+  ContestWriteInTally,
+  LogoMark,
+  Prose,
+  ReportSection,
+  TallyReport,
+  TallyReportColumns,
+  TallyReportMetadata,
+  Text,
+} from '@votingworks/ui';
+import {
+  Election,
+  getLabelForVotingMethod,
+  PartyIdSchema,
+  unsafeParse,
+  VotingMethod,
+  ContestId,
+} from '@votingworks/types';
+import { find } from '@votingworks/utils';
+import React, { forwardRef } from 'react';
+
+export interface Props {
+  batchId?: string;
+  batchLabel?: string;
+  election: Election;
+  writeInCounts: Map<ContestId, Map<string, number>>;
+  generatedAtTime?: Date;
+  isOfficialResults: boolean;
+  partyId?: string;
+  precinctId?: string;
+  scannerId?: string;
+  votingMethod?: VotingMethod;
+}
+
+// eslint-disable-next-line react/display-name
+export const ElectionManagerWriteInTallyReport = forwardRef<
+  HTMLDivElement,
+  Props
+>(
+  (
+    {
+      batchId,
+      batchLabel,
+      election,
+      writeInCounts,
+      generatedAtTime = new Date(),
+      isOfficialResults,
+      partyId: partyIdFromProps,
+      precinctId: precinctIdFromProps,
+      scannerId,
+      votingMethod,
+    },
+    ref
+  ) => {
+    const statusPrefix = isOfficialResults ? 'Official' : 'Unofficial';
+
+    const ballotStylePartyIds =
+      partyIdFromProps !== undefined
+        ? [unsafeParse(PartyIdSchema, partyIdFromProps)]
+        : Array.from(new Set(election.ballotStyles.map((bs) => bs.partyId)));
+
+    const precinctIds =
+      precinctIdFromProps === 'all'
+        ? election.precincts.map((p) => p.id)
+        : [precinctIdFromProps];
+
+    const writeInMetadataFooter = (
+      <React.Fragment>
+        <br />
+        <Text small as="span">
+          Note: Write-in votes adjudicated to official candidates are included
```

Comment:
I feel like it should either be "and not this report" or "and are not in this report"